dev-environment/nginx/pmml/static/boolean

```diff
@@ -0,0 +1 @@
+true
```

Comment:
What are those single line files used for?

Reply:
they are static files seved as json data for tests purpose (instead of creating dedicated server for tests that would serve json data we serve static files that look like the json and behave like them)